utils/__init__.py

```diff
@@ -79,6 +87,47 @@ def push_docker_image_to_repository(
     if username is not None and password is not None:
         docker_push_kwags["auth_config"] = {"username": username, "password": password}
     try:
-        docker_client.images.push(**docker_push_kwags)
+        # TODO better logs for docker push
+        for line in docker_client.images.push(
+            **docker_push_kwags, stream=True, decode=True
+        ):
+            print(line)
```

Comment:
👍🏼

Reply:
https://docker-py.readthedocs.io/en/stable/api.html#docker.api.image.ImageApiMixin.push

Reply:
just as shown in the link aarnphm posted (thanks mate!) it was a way for me to see the logs for the docker push since that was what takes the most time for me. But with sam build, this won't be necessary.